santoku/aws/test_s3_handler.py

```diff
@@ -177,9 +180,18 @@ def test_paginate(
         args = {
             "PaginationConfig": {"MaxItems": 1},
         }
-        with pytest.raises(exceptions.ParamValidationError) as e:
+        error_message = "Bucket argument is required for s3 paginator."

```

Comment:
Is the pytest error message match case sensitive? This one spells s3 while the next one spells S3 (pretty sure that is the correct spelling).

Reply:
It's not, but you're right, s3 in this message should be in uppercase.